codegen/src/lintdoc.rs

```diff
@@ -1515,9 +1544,19 @@ impl FromStr for CodeBlockTest {
             line_count: 0,
             hidden_lines: vec![],
             options_code_block: None,
+            file_path: None,
         };
 
         for token in tokens {
+            if let Some(file) = token.strip_prefix("file=") {
+                if file.is_empty() {
+                    bail!("The 'file' attribute must be followed by a file path");
+                }
+
+                test.file_path = Some(file.to_string());
```

Comment:
This assumes that the code examples have only one file. If the code examples use multiple files, is that an error? 

If that's an error, do we catch that in the main repository?

If it's not an error, we should this be a Vector?

Reply:
I think this logic gets invoked _per code block_, no? A single code block always represents a single file, but there can be multiple code blocks in an example.

Reply:
yeah each one of these tests is for a single code block, initialize [here](https://github.com/biomejs/website/blob/9b63a722214d46e2e4e200e234476f2809e7ce4d/codegen/src/lintdoc.rs#L1223) in the code start event and evaluated [here](https://github.com/biomejs/website/blob/9b63a722214d46e2e4e200e234476f2809e7ce4d/codegen/src/lintdoc.rs#L1246) in the end code block event